openassessment/workflow/models.py

```diff
@@ -320,7 +326,9 @@ def update_from_assessments(self, assessment_requirements):
                 # Update the assessment_completed_at field for all steps
                 # All steps are considered "assessment complete", as the staff score will override all
                 for step in steps:
-                    step.assessment_completed_at=now()
+                    step.assessment_completed_at = now()
+                    if override_submitter_requirements:
+                        step.submitter_completed_at = now()
```

Comment:
These 2 additional lines are the only functional (not quality-checker-based) changes in this file.